Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -87,6 +101,33 @@ script:
     ''' HELPER FUNCTIONS '''
 
 
+    def is_enough_preferred_vendors(scan_results):
+        if not (PREFERRED_VENDORS and PREFERRED_VENDORS_THRESHOLD):
+            return False
+        if PREFERRED_VENDORS and not PREFERRED_VENDORS_THRESHOLD:
+            return_error("Error: If you entered Preferred Vendors you must also enter Preferred Vendors Threshold")
+        if "scans" not in scan_results:
+            return False
+
+        counter = 0
+
+        vendors_scans_dict = scan_results["scans"]
+        list_of_preferred_vendors = PREFERRED_VENDORS.split(',')
+
+        for i in range(len(list_of_preferred_vendors)):
```

Comment:
Use comprehension.
`.strip()` each element to handle case of `, ` or similar separator.